commands.go

```diff
@@ -20,6 +20,11 @@ func runCommands(line string, u *user, isSlack bool) {
 	if line == "" {
 		return
 	}
+	if u.messaging != nil && !(strings.HasPrefix(line, "./") || strings.HasPrefix(line, "=")) {
```

Comment:
Side effect of having `strings.HasPrefix(line, "./")` in this condition is that if I send `./help` while in a private chat dm, it will be broadcast to a whole room. Fix by just removing that part of the condition:
```suggestion
	if u.messaging != nil && !strings.HasPrefix(line, "=") {
```
Alternatively, move this block after the check for whether the line is a DM (better design IMO)